### PR TITLE
OpenVR: upgrade to version 1.0.6, for better generic tracker support

### PR DESCRIPTION
**Pull request by hyperlogic**
QA Test:  smoke test on a Vive

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 42262aac78843dc9a8f7e58445f42471fd055b2a available at: https://gist.github.com/bdc4ff6767d93844980d734b8da13705<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10145-42262aa.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10145-42262aa.dmg

File: plugins/openvr/src/ViveControllerManager.cpp [hunk 1]
```diff
@@ -140,9 +140,15 @@ void ViveControllerManager::InputDevice::update(float deltaTime, const controlle
...
     handleHandController(deltaTime, rightHandDeviceIndex, inputCalibrationData, false);
 
-    // collect raw poses
+    // collect poses for all generic trackers
     for (int i = 0; i < vr::k_unMaxTrackedDeviceCount; i++) {
-        handleTrackedObject(i, inputCalibrationData);
+        if (_system->GetTrackedDeviceClass(i) == vr::TrackedDeviceClass_GenericTracker) {
+            handleTrackedObject(i, inputCalibrationData);
+        } else {
+            uint32_t poseIndex = controller::TRACKED_OBJECT_00 + i;
```
Comment (ZappoMan) [inline plugins/openvr/src/ViveControllerManager.cpp]: wouldn't it make more sense to put this code in `handleTrackedObject()`?

Comment (hyperlogic) [inline plugins/openvr/src/ViveControllerManager.cpp]: Sure, I can do that.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4e49774d0efff81324ee6d13171a379089aaafeb available at: https://gist.github.com/bdc4ff6767d93844980d734b8da13705<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10145-4e49774.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR10145-4e49774.dmg

**Comment by sethalves** [comment]
I tried this PR on vive (and an oculus, just for good measure).  it seems the same amount broken as master.

**Pull request merged**